+exp/Contents.m

```diff
@@ -0,0 +1,53 @@
+% +EXP Classes and functions for the Rigbox Experiment framework
+%   The Experiment framework provides a number of features for stimuli-
+%   delivering experiments with a trial structure.  The framework also
+%   provides a way to parameterize individual experiments and trials.
+%   Visual and auditory stimuli can be controlled by experiment phases or
+%   by a Signals reactive network.  Phases changes are managed by an
+%   event-handling system.
+%
+% Files
```

Comment:
nice, I like this organization. A short one to two sentence description of each header would be helpful here